index.html

```diff
@@ -13,6 +13,50 @@
   </head>
 
   <body>
-   
+    <div class="container">
+      <header>
+        <div class="header">
+          <div class="header-p">
+            <p class="name">Kevin Toro</p>
+          </div>
+          <div class="fa-bars-icon">
+            <i class="fa-solid fa-bars"></i>
+          </div>
+        </div>
+      </header>
+      <main>
+        <section class="section-1">
+          <div class="headline">
+            <div class="Primary-text">Hey There. I’m Toro</div>
+            <div class="Supporting-text">I’m a Software Developer</div>
```

Comment:
This would also go in some `hx` tag. Since One page should only contain 1 `h1` tag. So this should be encapsulated in a `h2` tag, again for semantic purposes.